docs/en/stack/ml/df-analytics/dfa-classification.asciidoc

```diff
@@ -5,55 +5,51 @@
-outputs; one for every possible music genre. In the {version} version of the 
-{stack}, you can perform only binary {classanalysis}.
+{reganalysis} which predicts continuous, numerical values. The maximum number of 
+classes for {classanalysis} is 30.
 
 
 [discrete]
-[[dfa-classification-features]]
-==== {feature-vars-cap}
+[[dfa-classification-depvar]]
+==== {depvar-cap}
 
-When you perform {classification}, you must identify a subset of fields that you 
-want to use to create a model for predicting another field value. We refer to 
-these fields as _{feature-vars}_ and _{depvar}_, respectively. 
-{feature-vars-cap} are the values that the {depvar} value depends on. There are 
-three different types of {feature-vars} that you can use with our 
-{classification} algorithm: numerical, categorical, and boolean. Arrays are not 
-supported in the {feature-var} fields.
+When you create a {classification} job, you must specify which field contains 
+the classes that you want to predict. We refer to this field as {depvar}. The 
+rest of the fields that are included in the analysis are used to create the 
+model for predicting the value of the {depvar} (these are the {feature-vars}). 
+All {ref}/put-dfanalytics.html#dfa-supported-fields[supported fields] are 
+included by default, however, you can explicitly exclude and/or include fields. 
+Since the runtime and the necessary resources for the analysis are increasing as 
+the number of included fields is growing, consider to include only relevant 
+fields. For more information about field selection, see 
+{ref}/explain-dfanalytics.html[Explain data frame analytics API].
 
 
 [discrete]
 [[dfa-classification-supervised]]
 ==== Training the {classification} model
 
 {classification-cap} – just like {regression} – is a supervised {ml} process. It 
-means that you need to supply a labeled training dataset that has some 
-{feature-vars} and a {depvar}. The {classification} algorithm learns the 
-relationships between the features and the {depvar}. Once you’ve trained the 
-model on your training dataset, you can reuse the knowledge that the model has 
-learned about the relationships between the data points to classify new data. 
-Your training dataset should be approximately balanced which means the number of 
-data points belonging to the various classes should not be widely different, 
-otherwise the {classanalysis} may not provide the best predictions. Read 
-<<dfa-classification-imbalanced-classes>> to learn more.
+means that you need to supply a labeled training data set that has a {depvar} 
+and some fields that are related to it. The {classification} algorithm learns 
+the relationships between these fields and the {depvar}. Once you’ve trained the 
+model on your training data set, you can reuse the knowledge that the model has 
+learned about the relationships between the data points to classify new data.
+
+Although the effects of imbalanced data are automatically mitigated before the 
```

Comment:
Not mandatory, but I think it would be more impactful to split this sentence.
```suggestion
The effects of imbalanced data are automatically mitigated before the 
```